aws-py-fargate/README.md

```diff
@@ -121,26 +121,26 @@ Next, to deploy the application and its infrastructure, follow these steps:
 
     Notice that `pulumi up` redeploys just the parts of the application/infrastructure that you've edited.
 
-    ```
```

Comment:
Do these snippets render “better” with `bash`?

Reply:
Not sure if it renders better, but it stops the markdown linter from complaining; and since the prompt is typically bash, I went with bash.